src/test/java/cloud/fogbow/ras/core/plugins/interoperability/azure/util/AzureImageOperationUtilTest.java

```diff
@@ -70,5 +92,76 @@ public void testBuildAzureVirtualMachineImageBySuccessfully() {
         Assert.assertEquals(skuExpected, azureVirtualMachineImage.getSku());
     }
 
+    // test case: When calling encodeImageId method with mocked methods,
+    // it must verify if it returns the a new encoded id
+    @Test
+    public void testEncodeImageIdSuccessfully() throws UnexpectedException {
+        // set up
+        String id = "publisher@#offer@#sku";
+
+        // exercise
+        String encodedId = AzureImageOperationUtil.encodeImageId(id);
+
+        // verify
+        Assert.assertNotEquals(encodedId, id);
+    }
+
+    // test case: When calling encodeImageId method and a UnsupportedEncodingException
+    // occurs, it must verify if it rethrow a UnexpectedException
+    @Test
+    public void testEncodeImageIdFail() throws UnsupportedEncodingException, UnexpectedException {
+        // set up
+        PowerMockito.mockStatic(URLEncoder.class);
+        String id = "publisher@#offer@#sku";
```

Comment:
Use the Separator constant.

Reply:
Use it the same idea in all test class, so you must check other tests.

Reply:
Done.